COMETwebapp/Components/SubscriptionDashboard/UsersSubscriptionTable.razor

```diff
@@ -0,0 +1,127 @@
+﻿<!------------------------------------------------------------------------------
+Copyright (c) 2022 RHEA System S.A.
+
+    Authors: Justine Veirier d'aiguebonne, Sam Gerené, Alex Vorobiev, Alexander van Delft
+
+    This file is part of COMET WEB Community Edition
+     The COMET WEB Community Edition is the RHEA Web Application implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+
+    The COMET WEB Community Edition is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Affero General Public
+    License as published by the Free Software Foundation; either
+    version 3 of the License, or (at your option) any later version.
+
+    The COMET WEB Community Edition is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see http://www.gnu.org/licenses/.
+------------------------------------------------------------------------------->
+@inject IIterationService iterationService
+@inject ISessionAnchor ISessionAnchor
+
+
+<div>
+    @if(this.SubscribedParameters.Count == 0)
+    {
+        <div>
+            You don't have any subscribed Parameters.
+        </div>
+    } else
+    {
+        <DxDataGrid Data="@ViewModelSubscribedParameters" PageSize="0">
+            <DxDataGridColumn Field="@nameof(ParameterSubscriptionViewModel.ElementName)" Caption="Element" TextAlignment="DataGridTextAlign.Center" SortOrder="DataGridColumnSortOrder.Ascending" />
+            <DxDataGridColumn Field="@nameof(ParameterSubscriptionViewModel.ParameterType)" Caption="Parameter" TextAlignment="DataGridTextAlign.Center">
+                <DisplayTemplate>
+                    @if((context as ParameterSubscriptionViewModel).IsIncompleted == true)
```

Comment:
the `IsIncompleted` is not an english word, please use a different name for this property.